src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -960,17 +960,17 @@ public function query_child_records( $args = array() ) {
 	 *
 	 * @return WP_Query|WP_Error|bool
 	 */
-	public function get_child_record_by_status( $status = 'success', $qty = -1, array $args = array() ) {
+	public function get_child_record_by_status( $status = 'success', $qty = -1, array $args = [] ) {
 		$statuses = Tribe__Events__Aggregator__Records::$status;
 
 		if ( ! isset( $statuses->{ $status } ) && 'trash' !== $status ) {
 			return false;
 		}
 
-		$args = array_merge( $args, array(
+		$args  = array_merge( $args, [
 			'post_status'    => $statuses->{$status},
 			'posts_per_page' => $qty,
-		) );
+		] );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 2b19b620c0a37d42434aad878a737005</sub>